src/md/src/Analysis/AutoAnalysisOfLaunches.md

```diff
@@ -20,18 +20,16 @@ There are several ways to use an analyzer in our application:
 
 * Do not use any Analyzers at all and do an analytical routine by yourself;
 
-### ReportPortal Analyzer
-If you want to reduce time costs, you can deploy the  ReportPortal with a service Analyzer. For that you should add an info about this service in a docker compose file and deploy service Analyzer.
+### ReportPortal Analyzer. How the Auto-Analysis is working
 
-**How the AUTO-ANALYSIS is working**
+If you want to reduce time costs, you can deploy the  ReportPortal with a service Analyzer. For that you should add an info about this service in a docker-compose file and deploy service Analyzer. Service ElasticSearch will be running together with Analyzer.
```

Comment:
i think we should re-phrase it. It's not really clear how to reduce the costs. 
I would say "ReportPortal's auto-analyzer allows users to reduce time spent on test execution investigation by analyzing test failures in automatic mode"

Reply:
Default analysis component is running along with ElasticSearch which is used for test logs indexing.